indra/newview/skins/default/xui/en/strings.xml

```diff
@@ -3818,6 +3818,7 @@ Please reinstall viewer from  https://secondlife.com/support/downloads/ and cont
 
   <string name="voice_morphing_url">https://secondlife.com/destination/voice-island</string>
   <string name="premium_voice_morphing_url">https://secondlife.com/destination/voice-morphing-premium</string>
+  <string name="no_voice_morphing_faq_url">https://lindenlab.freshdesk.com/support/solutions/articles/31000173560-webrtc-update-for-voice-chat-faq/</string>
```

Comment:
That URL doesn't exist https://lindenlab.freshdesk.com/support/solutions/articles/31000173560-webrtc-update-for-voice-chat-faq/ - or will it be created before this goes out ?

Reply:
I got no response from the webrtc-voice channel when I asked that question in slack, maybe @kylelinden knows?

Reply:
@kylelinden says the URL is accurate and the doc is "nearly done"